File: CursoProgramacionCero/condicionales.py
# Condiciones simples
edad = 18
if edad >= 18:
    print("Puedes votar")
else:
    print("No puedes votar")

# Condiciones anidadas
nota = 80
if nota >= 90:
    print("Aprobaste con una calificación de:", nota)
elif nota >= 80:
    print("Tienes un buen resultado con una calificación de:", nota)
else:
    print("Tienes un mal resultado con una calificación de:", nota)

# Switch
diaSemana = "Domingo"
if diaSemana == "Lunes":
    print("Hoy es Lunes")
elif diaSemana == "Martes":
    print("Hoy es Martes")
elif diaSemana == "Miércoles":
    print("Hoy es Miércoles")
elif diaSemana == "Jueves":
    print("Hoy es Jueves")
elif diaSemana == "Viernes":
    print("Hoy es Viernes")
elif diaSemana == "Sábado":
    print("Hoy es Sábado")
elif diaSemana == "Domingo":
    print("Hoy es Domingo")
else:
    print("Día inválido")

File: CursoProgramacionCero/Condicionales/condicionales.js
// Condiciones simples
let edad = 18;

if (edad >= 18) {
    console.log("Puedes votar");
} else {
    console.log("No puedes votar");
}

// Condiciones anidadas
let nota = 80;

if (nota >= 90) {
    console.log("Aprobaste con una calificación de:", nota);
} else if (nota >= 80) {
    console.log("Tienes un buen resultado con una calificación de:", nota);
} else {
    console.log("Tienes un mal resultado con una calificación de:", nota);
}

// Switch
let diaSemana = "Domingo";

switch (diaSemana) {
    case "Lunes":
        console.log("Hoy es Lunes");
        break;
    case "Martes":
        console.log("Hoy es Martes");
        break;
    case "Miércoles":
        console.log("Hoy es Miércoles");
        break;
    case "Jueves":
        console.log("Hoy es Jueves");
        break;
    case "Viernes":
        console.log("Hoy es Viernes");
        break;
    case "Sábado":
        console.log("Hoy es Sábado");
        break;
    case "Domingo":
        console.log("Hoy es Domingo");
        break;
    default:
        console.log("Día inválido");
}

